Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -205,16 +213,38 @@ class BranchStopper {
             return config.hasCuts(eta) ? &config.getCuts(eta) : nullptr;
           }
         },
-        *m_config);
+        *m_cfg.trackSelectorCfg);
 
     if (singleConfig == nullptr) {
       ++m_nStoppedBranches;
       return BranchStopperResult::StopAndDrop;
     }
 
+    bool tooManyHolesPS = false;
+    if (!(m_cfg.pixelVolumes.empty() && m_cfg.stripVolumes.empty()) &&
+        !(m_cfg.maxPixelHoles == std::numeric_limits<std::size_t>::max() &&
+          m_cfg.maxStripHoles == std::numeric_limits<std::size_t>::max())) {
```

Comment:
Right. I wanted to avoid any time taken if the new feature is not used, but we could just check the two sets and not check the max holes. Is that a good compromise?

Reply:
Sounds good 👍